### DISCUSSION TITLE
Checkbox for ModC=ModA

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#230] [Ideas]
I really need an option to always keep Model С = Model A for any Add difference. It gives good results for some cases.

**Comment by miasik** [comment]
Well, the workaround #242 almost solved my need.
I still need something(a button?) to clear the field and set ModC as undefined if ModC was previously selected.

Anyway, I think that a checkbox would be better.

**Comment by miasik** [comment]
@wkpark 
Could you please add a button to reset Model С?

**Reply by wkpark** [reply]
It's hard to think of a suitable UI.

Here's what I added without breaking the current UI. (2-methods)
- add "Add difference (Model_A)" or shortly "Add diff (Model A)"
- add a "Model_A" in the dropdown menu.

![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/232347/2b50ab68-ccee-48bc-9084-1ed0a54aa0f1)

**Reply by miasik** [reply]
Well, the 2nd method looks better for me too. Thank you!

For now, because of #242, I can use ",somemodel1,somemodel2" for model_c in XYplot and SM sets model_c as model_a for the first step. Hope it'll continue working the same way.
